client/app/dedicated/server/interfaces/interfaces.html

```diff
@@ -0,0 +1,200 @@
+<h1 data-translate="dedicated_server_interfaces_title"></h1>
+<p data-translate="dedicated_server_interfaces_introduction"></p>
+<p>
+    <span data-translate="dedicated_server_interfaces_vrack_information"></span>
+    <a href="https://docs.ovh.com" data-translate="dedicated_server_interfaces_vrack_information_link_label"></a>
```

Comment:
Should be in a constant file

Reply:
We don't know for now wich guide to link. It's just for the mockup staging.
It'll be updated